tests/sepp/Workloads/IterateDocuments.cpp

```diff
@@ -105,11 +111,12 @@ void IterateDocuments::Thread::run() {
   std::unique_ptr<rocksdb::Iterator> it(rootDB->NewIterator(ro, docCF));
 
   OperationOptions options;
-  std::uint64_t numIts = 0;
-  for (it->Seek(bounds.start()); it->Valid(); it->Next(), ++numIts) {
-    // TODO
+  for (it->Seek(bounds.start()); it->Valid(); it->Next()) {
+    ++_operations;
+    if (_operations % 512 == 0 && execution().stopped()) {
```

Comment:
```suggestion
    if (_operations % 512 == 0 && (execution().stopped() || shouldStop())) {
```
Alternatively the `execution().stopped()` check could be integrated in `shouldStop()` (I would actually prefer that).

Reply:
Done